package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "db:migration": "cross-env NODE_ENV=test sequelize db:migrate",
     "undo:migration": "cross-env NODE_ENV=test sequelize db:migrate:undo:all",
+    "test-dev": "NODE_ENV=test sequelize db:migrate:undo:all && NODE_ENV=test sequelize db:migrate && npm test",
```

Comment:
Why did you remove our `test` script? please check the development branch to see what we have.